### PR TITLE
Ports: Add port aria2 version 1.36.0

### PR DESCRIPTION
**Pull request by EWouters**
My previous PR (https://github.com/SerenityOS/serenity/pull/13665) was closed because it was a while ago I worked on this, but it sort of works now. It was able to download a torrent, but http downloads fail.

Thanks for fixing the number parsing issue timschumi! (https://github.com/SerenityOS/serenity/pull/13665#pullrequestreview-1010297521)  

image

### CODE REVIEW

File: Ports/aria2/package.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='aria2'
+version='1.36.0'
+files="https://github.com/aria2/aria2/releases/download/release-${version}/aria2-${version}.tar.xz ${port}-${version}.tar.xz 58d1e7608c12404f0229a3d9a4953d0d00c18040504498b483305bcb3de907a5"
+auth_type="sha256"
```
Comment (linusg) [inline Ports/aria2/package.sh:5]: single quotes here

File: Ports/aria2/package.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+depends=("libssh2" "libxml2" "openssl" "zlib" "libuv")
```
Comment (linusg) [inline Ports/aria2/package.sh:6]: Please break this into one per line

**Review by linusg — CHANGES_REQUESTED** [review]

File: Ports/aria2/package.sh (hunk 2 above)
Comment (EWouters) [reply Ports/aria2/package.sh:6]: Are you sure? I can't see any other package.sh scripts that do that.

Comment (linusg) [reply Ports/aria2/package.sh]: Yes, it creates more meaningful diffs/git blame results :)

**Pull request merged**